cmd/manager/main.go

```diff
@@ -100,7 +100,7 @@ func startListener(ctx context.Context, ha host.Host, listenPort int, insecure b
 	logger.Info().Msgf("I am %s", fullAddr)
 
 	ha.SetStreamHandler("/b7s/manager/1.0.0", func(s network.Stream) {
-		if allowedPeer == "" && s.Conn().RemotePeer().Pretty() != allowedPeer {
+		if allowedPeer == "" && s.Conn().RemotePeer().String() != allowedPeer {
```

Comment:
Didn't see this in the manager code before.

This part looks fishy - `if allowedPeer == ""`.
Shouldn't that be an `!=`?

a) if the allowedPeer value is set (not empty) AND
b) remote peer is not that peer =>

don't process the message.
Does this work right now if the allowedPeer is not set?

Reply:
yeah this works, I think the label is probably misleading, as it's the, I'm bad at naming things, sue me. :)